content/conference/2021_online_coding_sprint.md

```diff
@@ -10,19 +10,35 @@ homepage = "https://discuss.haiku-os.org/t/haiku-coding-sprint-2021/11327"
 country = "Worldwide"
 +++
 
-Since it is not so easy to organize in-person events currently, this year
-coding sprint will be online.
+This year Haiku will be holding a virtual coding sprint (due to international travel restrictions), between 18th and the 31st of October. These dates were chosen because they matched up with the traditional dates for the in-person coding sprint and the BeGeistert conference.  
 
-In order to accomodate everyone, it will last for two weeks, and you are free
-to join (or leave) at any time.
+This also lines up with [Hacktoberfest](https://hacktoberfest.digitalocean.com/) - do note though that whilst working on Haiku, the operating system for Hacktoberfest won't work (the development work for this happens on our own hosted git repositories and on Gerrit, not on GitHub), helping out at [HaikuArchives](https://github.com/HaikuArchives), [HaikuPorts](https://github.com/haikuports/haikuports) or doing some work on our [website](https://github.com/haiku/website) and [web infrastructure](https://github.com/haiku/infrastructure) will as those are hosted on GitHub. 
 
-The event is open to everyone who wants to work on Haiku or any related software
-project: write new applications, port existing applications, update already
-ported applications, no matter if it's your own or someone else's. And of course
-it's also possible to dig deeper into Haiku internals and fix some bugs there.
+## How does the coding sprint work?
+In past years, coding sprints involved gathering developers in a room (a meeting room, usually) where they would work on Haiku. Obviously, with this virtual coding sprint everything will run online, presumably with an IRC channel or another online platform.  
 
-The event overlaps with Hacktoberfest, so you can also take part into that if
-you want to.
+Anyone can participate, and there is **no obligation on how much time you put in** even though the sprint runs for two weeks. It's a good opportunity for existing contributors to work on something Haiku-related, like fix bugs or embark on that large project that's been sitting on their to-do list for a while, or help resurrect an app (might be their own or an app from the BeOS days sitting around in [HaikuArchives waiting for some TLC](https://github.com/HaikuArchives)), help finalise patches waiting on [Gerrit](https://review.haiku-os.org), submit proposals, etc. etc. And since there'll be more people online during this sprint, you'll have plenty of help from your fellow developers on hand!
+
+## What about new contributors?
+This is also a good chance for new contributors to get started too! Just to ensure you're ready for the sprint, it's a good idea to first check out our [page on how you can get started on development](https://www.haiku-os.org/community/getting-involved/developing), with ideas on what you might like to work on. You might also want to:
+* [Read the development FAQ](https://www.haiku-os.org/development/faq)
```

Comment:
You are drowning the reader with a lot of more-or-less relevant information here. The "getting started" page should already point to all these things, there is no need to repeat it all here.